dbt/models/ferc1/out_ferc1__yearly_steam_plants_sched402/schema.yml

```diff
@@ -63,8 +71,12 @@ sources:
           - name: plant_type
           - name: record_id
           - name: water_limited_capacity_mw
-
 models:
   - name: validate_ferc1__yearly_steam_plants_sched402
+    description: >
+      This is a validation model which takes the above table as input and computes
+      values more convenient for testing, including:
+
+      - capacity ratios; not currently tested
```

Comment:
If the capacity ratios are not being tested, why do we have this?

Reply:
@marianneke @zschira this model was one of your additions; do you know?